amivapi/settings.py

```diff
@@ -165,26 +165,30 @@
     'termsOfService': 'todo',
     'contact': {
         'name': 'AMIV an der ETH',
-        'url': 'https://amiv.ch/'
+        'url': 'https://amiv.ethz.ch/'
     },
     'license': {
         'name': 'GNU Affero General Public License',
         'url': 'https://github.com/amiv-eth/amivapi/blob/master/LICENSE',
-    }
+    },
+}
+
+SWAGGER_LOGO = {
+    # TODO: Replace with AMIVAPI logo endpoint (or similar)
```

Comment:
TODO should not be in code. Create an issues instead.

Reply:
I can remove the TODO from the comment, if you think thats more appropriate.

Reply:
To keep track of issues and enhancements, the issues should be used to have a searchable collection of items in one place. Having this TODO in the code might produce inconsistent situations.

It might be possible that we decide to keep it like that. There may be a discussion in the said issue about that and the issue might be closed. The TODO still exists in the code, but there is no intention to do it anymore.